web/src/app/schedules/temp-sched/TempSchedAddNewShift.tsx

```diff
@@ -227,13 +239,11 @@ export default function TempSchedAddNewShift({
                   // value held in form input
                   mapValue={(nextVal: string, formValue: Value) => {
                     const nextValDT = DateTime.fromISO(nextVal, { zone })
-                    if (!formValue || !nextValDT.isValid) return ''
-                    return nextValDT
-                      .diff(
-                        DateTime.fromISO(formValue.start, { zone }),
-                        'hours',
-                      )
-                      .hours.toString()
+                    const formValDT = DateTime.fromISO(formValue?.start, {
```

Comment:
```suggestion
                    const formValDT = DateTime.fromISO(formValue?.start ?? '', {
```

I'm not sure if luxon needs you to handle the undefined case here but it's being handled like this in other places.